.travis.yml

```diff
@@ -56,7 +56,7 @@ script:
   - RUNNING_CONTAINERS=`docker ps | wc -l`;
   - if [ "${RUNNING_CONTAINERS//[[:space:]]/}" == "1" ]; then echo "[E] Container is not started\!"; docker logs prm --details --since 5h; exit 1; fi;
   # Run acceptance tests on Docker container
-  - "MIX_TEST_HOST=localhost MIX_TEST_PORT=4000 mix test test/acceptance"
+  - MIX_TEST_HOST=localhost MIX_TEST_PORT=4000 mix test test/acceptance
```

Comment:
this is Acceptance test running command. That's what pull request was created for